src/lib/y2users/user_validator.rb

```diff
@@ -47,21 +48,21 @@ def issues(skip: [])
       list = Y2Issues::List.new
 
       if !skip.include?(:name)
-        err = Yast::UsersSimple.CheckUsernameLength(user.name)
+        err = check_length
         add_fatal_issue(list, err, NAME_LOC)
 
-        err = Yast::UsersSimple.CheckUsernameContents(user.name, "")
+        err = check_characters
         add_fatal_issue(list, err, NAME_LOC)
 
         # Yast::UsersSimple.CheckUsernameConflicts is currently used only when manually creating
```

Comment:
Is the comment still valid?

Reply:
yes, it still applies, as I just duplicate userssimple functionality in context of y2users, so installation and autoinstallation for local users. So it is more like note when we want to check duplicities in running system that it has same limitation that checks against hard coded list.